doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/AptVerifier.java

```diff
@@ -59,8 +60,8 @@ public void verify( String file )
         //
         // ----------------------------------------------------------------------
 
-        HtmlDivision div = (HtmlDivision) elementIterator.next();
-        assertEquals( "section", div.getAttribute( "class" ) );
+        /* cannot use HtmlSection until https://sourceforge.net/p/htmlunit/bugs/1961/ is fixed */
```

Comment:
The issue seems to be closed, has this workaround to remain until the next version of HtmlUnit?

Reply:
Please createa a folloup issue on JIRA for this not be forgotten.

Reply:
I see that https://sourceforge.net/p/htmlunit/bugs/1961/ was closed roughly 24 hours ago, so tried to give a go upgrading to the latest httpunit.

This is the point I got stuck with the enforcer plugin telling me that doxia contains 1.7 bytecode, so I came to a screeching halt.

It has jogged a memory - as I recall I tried upgrading htmlunit when I encountered the above bug, and versions of htmlunit newer that v2.16 from 2015 require java7+, which is failed by maven's enforcer config.

I don't think this can be fixed until maven releases a new parent POM supporting a minimum of java8. Let me raise a jira ticket for this.

Reply:
We have bumped to Java 7, is an update now possible?

Reply:
Let me try and see - I'll find the htmlunit release just before they jumped to java8 and try that.